mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -606,13 +672,28 @@
     </CAbilMorph>
     <CAbilMorph id="MorphToLurker">
         <Flags index="IgnoreFacing" value="1"/>
+        <Flags index="WaitUntilStopped" value="0"/>
+        <InfoArray index="0" RandomDelayMax="0"/>
     </CAbilMorph>
     <CAbilMorph id="MorphToOverseer">
         <Flags index="IgnoreFacing" value="1"/>
         <Flags index="WaitUntilStopped" value="0"/>
     </CAbilMorph>
     <CAbilMorph id="MorphToRavager">
         <Flags index="IgnoreFacing" value="1"/>
+        <Flags index="WaitUntilStopped" value="0"/>
+        <InfoArray index="0" RandomDelayMax="0"/>
```

Comment:
**undocumented**: Hydralisk -> Lurker Morph's random starting delay was changed from [0, 0.5] to [0, 0]

This could have been required for the morph's cancel. Ravager has no random delay as well (like Warp Prism and Liberator).

Should this be done for other units as well? E.g. Vikings, Thors, Widow Mine, Siege Tank, Hellion?